pyconcz/templates/default/admin/common/phase_values.html

```diff
@@ -1,27 +1,39 @@
-<table class="results">
-  <tr>
-    <th></th>
-    <th>Name</th>
-    <th>Active since</th>
-  </tr>
-{% for value in values %}
-  <tr>
-    <td>
-      <small>
-      {% if value == current_value %}→{% endif %}
-      {% if not value.enabled %}×{% endif %}
-      </small>
-    </td>
-    <td>
-      {% if value == current %}
-        <strong>{{ value }}</strong>
-      {% else %}
-        <small>{{ value }}</small>
-      {% endif %}
-    </td>
-    <td>
-      <small>{{ value.active_since | date:'Y-m-d H:i' }}</small>
-    </td>
-  </tr>
-{% endfor %}
+{% load static %}
+<table class="results" style="margin:0 0 -9px;border:0;">
+    <thead>
+    <tr>
+        <th style="background-color: transparent;border-top: 0;padding-bottom: 8px;width: 5%;"></th>
+        <th style="background-color: transparent;border-top: 0;padding-bottom: 8px;width: 20%;padding-left: 8px;">Name</th>
+        <th style="background-color: transparent;border-top: 0;padding-bottom: 8px;width: 75%;padding-left: 8px;">Active since</th>
+    </tr>
+    </thead>
+    <tbody>
+    {% for value in values %}
+        <tr>
+            <td style="text-align: right;">
+                <small>
+                    {% if value == current_value %}
+                        <img src="data:image/svg+xml;utf8,%3Csvg xmlns='http://www.w3.org/2000/svg' width='13' height='13' viewBox='0 0 13 13'%3E%3Cpath fill='%23447e9b' d='M6.5.93A5.57 5.57 0 0 0 .93 6.5a5.57 5.57 0 0 0 5.72 5.57 5.57 5.57 0 0 0 5.42-5.57A5.57 5.57 0 0 0 6.65.93zm.05 1.84c.12 0 .23.04.33.13l3.28 3.27c.16.18.17.45 0 .65L6.88 10.1a.48.48 0 0 1-.65 0l-.66-.66a.47.47 0 0 1 0-.65l1.37-1.36H3.31a.48.48 0 0 1-.46-.47v-.92c0-.24.2-.45.46-.47h3.63L5.57 4.21a.48.48 0 0 1 0-.65l.66-.66a.47.47 0 0 1 .32-.13z'/%3E%3C/svg%3E" alt="current value" />
+                    {% elif not value.enabled %}
```

Comment:
This means only `→` will show when a phase is overridden to a  disabled value. Previously, both `×` and `→` were there.
Is that what you want?

Reply:
Oh, didn't think it should be possible to make current value from a value that is not active even by overriding. But ok.